igor-web/src/main/groovy/com/netflix/spinnaker/igor/pubsub/PubsubSubscribers.groovy

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.pubsub
+
+import com.netflix.spinnaker.igor.model.PubsubType
+
+class PubsubSubscribers {
+  private Map<String, PubsubSubscriber> subscriberByName = new HashMap<String, PubsubSubscriber>()
```

Comment:
Still on the fence with respects to storing these as a map.

Alternative might be:
```
private Collection<NamedPubsubSubscriber> subscribers

void add(NamedPubsubSubscriber subscriber)
Collection<NamedPubsubSubscriber> getSubscribers(PubsubType pubsubType)
NamedPubsubSubscriber getByName(String name)
```

I generally find it easier to filter and manipulate lists rather than maps where you might need to look at both the key and value.

My feelings aren't terribly strong so I won't be upset if you keep this `Map`-based.